api/main.py

```
@@ -0,0 +1,167 @@
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+# Add parent directory to path (for services imports)
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from config.settings import settings
+from core.logging_config import get_logger
+
+logger = get_logger("api_server")
+
+# Import routes (legacy - being disabled systematically)
+# LEGACY DISABLED: from routes import job, user, files, upload, agents, download, workflow
+from routes import websocket, system  # Keep WebSocket + system (for Quick Actions)
```

Ruff (0.12.2)

`routes.system` imported but unused

Remove unused import: `routes.system`

(F401)